PPL/costAnalisis.md

```diff
@@ -0,0 +1,1724 @@
+   2. Costes derivados de producción
+   3. Costes de licencias en producción
+   4. Coste de marketing en producción
+   5. Amortización en producción
+   6. Desglose total de los costes de producción
+3. TCO
+4. ROI
+
+## Registro de cambios
+<table>
+  <tr>
+   <td><strong>Versión</strong>
+   </td>
+   <td><strong>Fecha</strong>
+   </td>
+   <td><strong>Descripción</strong>
+   </td>
+   <td><strong>Redacción</strong>
+   </td>
+   <td><strong>Revisión</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>1.0.0</td>
+   <td>1/2/2024</td>
+   <td>Completado la primera versión del análisis del coste preliminar del proyecto.</td>
+   <td>MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>CLAUDIA GILABERT PRIETO</td>
+  </tr>
+  <tr>
+   <td>1.0.1</td>
+   <td>02/02/24</td>
+   <td>Añadido desglose por semana a los costes derivados del desarrollo, nuevas licencias y presupuesto de contingencia.</td>
+   <td>MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>MANUEL ORTIZ BLANCO</td>
+  </tr>
+  <tr>
+   <td>1.0.2</td>
+   <td>04/02/24</td>
+   <td>Se han desglosado los costes en función de las distintas fases del desarrollo del proyecto. Se ha calculado la amortización de los equipos informáticos. Se ha realizado el análisis de los TCO.</td>
+   <td>MANUEL ORTIZ BLANCO</td>
+   <td>PEDRO LOPEZ RUZ</td>
+  </tr>
+  <tr>
+   <td>1.0.3</td>
+   <td>13/02/24</td>
+   <td>Añadidos costes de márketing.</td>
+   <td>MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>MARTA INÉS GONZÁLEZ DIÁNEZ</td>
+  </tr>
+  <tr>
+   <td>1.0.4</td>
+   <td>14/02/24</td>
+   <td>Añadida más información al documento y el ROI</td>
+   <td>MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>MARTA INÉS GONZÁLEZ DIÁNEZ</td>
+  </tr>
+  <tr>
+   <td>1.0.5</td>
+   <td>1/04/24</td>
+   <td>Añadida más información respecto al TCO</td>
+   <td>FÉLIX ÁNGEL GUDIEL GÜEMES</td>
+   <td>MARTA INÉS GONZÁLEZ DIÁNEZ</td>
+  </tr>
+  <tr>
+   <td>1.0.6</td>
+   <td>1/04/24</td>
+   <td>Añadida más información respecto a la variación del TCO</td>
+   <td>FÉLIX ÁNGEL GUDIEL GÜEMES, MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>FRANCISCO JAVIER CALDERÓN RODRÍGUEZ</td>
+  </tr>
+  <tr>
+   <td>1.0.7</td>
+   <td>24/04/24</td>
+   <td>Añadida más información respecto a la variación del TCO</td>
+   <td>MIGUEL ANGEL ROMALDE DORADO</td>
+   <td>FÉLIX ÁNGEL GUDIEL GÜEMES, FRANCISCO JAVIER CALDERÓN RODRÍGUEZ</td>
+  </tr>
+</table>
+
+<br/>
+
+#
+
+<br/>
+
+# 1. Costes del periodo de desarrollo
+
+
+## 1.1. Coste salarial del equipo (desarrollo)
+
+En este apartado se desglosan los costes salariales del equipo de desarrollo que contempla desde la primera semana de vida del proyecto hasta su puesta en producción, en este caso, 15 semanas. Hay que añadir que los costes de Community Manager serían computados a un trabajador externo ya que ningún miembro del equipo trabaja bien ese conjunto. Es por esto último que en la segunda tabla no se tiene en cuenta los costes salariales del Community Manager al ser un trabajador externo.
```

Comment:
What do you mean by this? i dont understand the community manager section

Reply:
This was the last version, did not updated it, my bad it will be updated tomorrow!